docs/azure_arc_jumpstart/azure_arc_k8s/pf9/pf9_bareos/_index.md

```diff
@@ -0,0 +1,208 @@
+
+## Prerequisites
+
+* An Ubuntu 20.04 installed physical machine or VM. For this demonstration, we would be using single node kubernetes cluster installation. This machine would also act as our management host.
+
+* The [kubectl](https://platform9.com/learn/tutorials/kubectl) exe for cluster management locally. If you would want to have a seperate host for cluster management, follow the below steps to [install kubectl](https://kubernetes.io/docs/tasks/tools/install-kubectl-linux/#install-using-native-package-management) on it.
+
+  ```shell
+  sudo apt-get update
+  sudo apt-get install -y apt-transport-https ca-certificates curl
+  sudo curl -fsSLo /usr/share/keyrings/kubernetes-archive-keyring.gpg https://packages.cloud.google.com/apt/doc/apt-key.gpg
+  echo "deb [signed-by=/usr/share/keyrings/kubernetes-archive-keyring.gpg] https://apt.kubernetes.io/ kubernetes-xenial main" | sudo tee /etc/apt/sources.list.d/kubernetes.list
+  sudo apt-get update
+  sudo apt-get install -y kubectl
+  ```
+
+  > **Note:** The *kubectl* package is installed from Kubernetes repository, hence the Google Cloud public signing key need to be downloaded to enable the repository.
+
+  > **All PMK cluster nodes would have kubectl installed once cluster is deployed.**
+
+* [Azure CLI (az)](https://docs.microsoft.com/en-us/cli/azure/install-azure-cli?view=azure-cli-latest) version 2.15.0 and above. This cli tool would help to setup the Azure resources and connect the Kubernetes cluster to Azure Arc.
+
+* [Helm](https://helm.sh/docs/intro/install/) version 3+ , to install the Azure Arc agents on the cluster.
+
+* [Enable subscription](https://docs.microsoft.com/en-us/azure/azure-resource-manager/management/resource-providers-and-types#register-resource-provider) with the two resource providers for Azure Arc-enabled Kubernetes.
+
+  ```shell
+  az provider register --namespace Microsoft.Kubernetes
+  az provider register --namespace Microsoft.KubernetesConfiguration
+  az provider register --namespace Microsoft.ExtendedLocation
+  ```
+
+  Registration is an asynchronous process, and registration may take approximately 10 minutes. You can monitor the registration process with the following commands
+
+  ```shell
+  az provider show -n Microsoft.Kubernetes -o table
+  az provider show -n Microsoft.KubernetesConfiguration -o table
+  az provider show -n Microsoft.ExtendedLocation -o table
+  ```
+
+* Install the Azure Arc Kubernetes CLI extensions *connectedk8s* and *k8s-configuration* .
+
+  ```shell
+  az extension add --name connectedk8s
+  az extension add --name k8s-configuration
+  ```
+
+  Run below command to verify that it is installed.
+
+  ```shell
+  az extension list -o table
+  ```
```

Comment:
You already have this as part of your automation in _pf9_az_connect_k8s.sh_ automation